repo/packages/E/etcd/0/marathon.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "{{etcd.framework-name}}",
+    "container": {
+        "docker": {
+            "image": "{{etcd.docker-image}}",
+            "forcePullImage": true
+        },
+        "type": "DOCKER",
+        "volumes": []
+    },
+    "args": [],
+    "cpus": {{etcd.cpus}},
+    "mem": {{etcd.mem}},
+    "instances": 1,
+    "ports": [0, 0, 0],
+    "env": {
+      "FRAMEWORK_NAME":"{{etcd.framework-name}}",
+      "CLUSTER_SIZE":"{{etcd.cluster-size}}",
+      "MESOS_MASTER":"{{mesos.master}}",
+      "ZK_PERSIST":"{{etcd.zk}}",
+      "VERBOSITY":"{{etcd.logging-verbosity}}",
+      "AUTO_RESEED":"{{etcd.auto-reseed}}",
+      "RESEED_TIMEOUT":"{{etcd.reseed-timeout}}",
+      "DISK_LIMIT":"{{etcd.disk-limit}}",
+      "CPU_LIMIT":"{{etcd.cpu-limit}}",
+      "MEM_LIMIT":"{{etcd.mem-limit}}",
+      "WEBURI": "http://{{etcd.framework-name}}{{etcd.dns-suffix}}:{{etcd.admin-port}}/stats"
+    },
+    "healthChecks": [{
+      "protocol": "HTTP",
+      "path": "/healthz",
```

Comment:
got it - missed that part of the fine print

On Wed, Sep 23, 2015 at 3:53 AM, Tyler Neely <notifications@github.com>
wrote:

> In repo/packages/E/etcd/0/marathon.json
> <https://github.com/mesosphere/multiverse/pull/7#discussion_r40177931>:
>
> > +    "env": {
> > +      "FRAMEWORK_NAME":"{{etcd.framework-name}}",
> > +      "CLUSTER_SIZE":"{{etcd.cluster-size}}",
> > +      "MESOS_MASTER":"{{mesos.master}}",
> > +      "ZK_PERSIST":"{{etcd.zk}}",
> > +      "VERBOSITY":"{{etcd.logging-verbosity}}",
> > +      "AUTO_RESEED":"{{etcd.auto-reseed}}",
> > +      "RESEED_TIMEOUT":"{{etcd.reseed-timeout}}",
> > +      "DISK_LIMIT":"{{etcd.disk-limit}}",
> > +      "CPU_LIMIT":"{{etcd.cpu-limit}}",
> > +      "MEM_LIMIT":"{{etcd.mem-limit}}",
> > +      "WEBURI": "http://{{etcd.framework-name}}{{etcd.dns-suffix}}:{{etcd.admin-port}}/stats"
> > +    },
> > +    "healthChecks": [{
> > +      "protocol": "HTTP",
> > +      "path": "/healthz",
>
> "maxConsecutiveFailures": 0 does what I want, based on my understanding:
> communicate a problem to operators but don't actually restart anything.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/multiverse/pull/7/files#r40177931>.
>